packages/block-editor/src/components/list-view/style.scss

```diff
@@ -99,7 +100,7 @@
 		height: auto;
 		padding: ($grid-unit-15 * 0.5) $grid-unit-05 ($grid-unit-15 * 0.5) 0;
 		text-align: left;
-		color: $gray-900;
+		color: currentColor;
```

Comment:
What's the background for these changes? My guess is List View supporting dark backgrounds? I couldn't find a way to trigger that when testing. It may be worth checking how it complements the changes in [Colorize template parts and Reusable blocks](https://github.com/WordPress/gutenberg/pull/45473#top).

Reply:
Yeah, I'll remove from the PR for now. Initially the PR had the navigation tree in the sidebar as well and it was left out for a follow-up.